sigilharvest.lic

```diff
@@ -65,7 +65,7 @@ class SigilHarvest
     
     Flags.add('sigilharvest-found', @@found_patterns) #flag for when any sigil has been found
     Flags.add('sigilharvest-type', @@sigil_patterns) #stores the type of sigil that was found
-    Flags.add('sigilharvest-noactions', /^You are unable to perceive any opportunities for improving the sigil\./)
+    Flags.add('sigilharvest-noactions', /You are unable to perceive any opportunities for improving the sigi/)
```

Comment:
"sigi"?

Reply:
Whoops. I was removing the period at the end of the pattern and must've deleted the 'l' too.